BrowsingModeDetector.js

```diff
@@ -164,16 +164,40 @@ var WebkitBrowser = function (BrowsingModeDetector) {
   this.detectBrowsingMode = function (_executeUserCallback) {
     var self = this;
 
-    var callbackWhenWebkitRequestFileSystemIsON = function () {
+    var callbackWhenWebkitStorageQuotaNotLimited = function () {
       self.BrowsingModeDetector.setBrowsingInNormalMode();
       _executeUserCallback();
     };
 
-    var callbackWhenWebkitRequestFileSystemIsOFF = function () {
+    var callbackWhenWebkitStorageQuotaLimited = function () {
       self.BrowsingModeDetector.setBrowsingInIncognitoMode();
       _executeUserCallback();
     };
 
+    var checkStorageQuota = function () {
+      if (typeof navigator.storage != 'undefined' && 
+      typeof navigator.storage.estimate != 'undefined') {
+        navigator.storage.estimate().then(function(estimate) {
+          if (estimate.quota < 120000000){
```

Comment:
For the sake of clarity add the 120000000 value to an object variable. Look at lines 16-17.

Reply:
Forgot to fix that. The object variable has been implemented in latest commit, thanks